staging/src/k8s.io/kubectl/pkg/generate/versioned/service.go

```diff
@@ -21,10 +21,11 @@ import (
 	"strconv"
 	"strings"
 
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```

Comment:
don't need the `v1` :)

Reply:
it seems a results of running go imports with go-vim, strange. Thanks for spotting it